modules/migration-performance-metrics.adoc

```diff
@@ -0,0 +1,118 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc
+// * migration-toolkit-for-containers/troubleshooting-mtc.adoc
+
+[id="migration-performance-metrics_{context}"]
+= Performance metrics
+
+The Migration controller records a set of metrics and pulls it into on-cluster monitoring storage. You can query the metrics via PromQL for diagnosing migration performance issues. All metrics reset when the Migration controller manager container restarts.
+
+== Provided Metrics
+
+=== cam_app_workload_migrations
+
+This metric is a count of MigMigration CRs over time. It is useful for viewing alongside `mtc_client...` metrics to collate API request info with migration status changes. This metric is included in Telemetry.
```

Comment:
Mark up CRs correctly with backticks.

You will probably have to explain the "mtc_client" metrics. What is referred to in the elipses?

info > information. 

"Telemetry" - What does this mean?